pkg/query-service/auth/auth.go

```
@@ -299,39 +380,11 @@ func Login(ctx context.Context, request *model.LoginRequest) (*model.LoginRespon
 		return nil, err
 	}
 
-	accessJwtExpiry := time.Now().Add(JwtExpiry).Unix()
-
-	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
-		"id":    user.Id,
-		"gid":   user.GroupId,
-		"email": user.Email,
-		"exp":   accessJwtExpiry,
-	})
-
-	accessJwt, err := token.SignedString([]byte(JwtSecret))
-	if err != nil {
-		return nil, errors.Errorf("failed to encode jwt: %v", err)
-	}
-
-	refreshJwtExpiry := time.Now().Add(JwtRefresh).Unix()
-	token = jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
-		"id":    user.Id,
-		"gid":   user.GroupId,
-		"email": user.Email,
-		"exp":   refreshJwtExpiry,
-	})
-
-	refreshJwt, err := token.SignedString([]byte(JwtSecret))
-	if err != nil {
-		return nil, errors.Errorf("failed to encode jwt: %v", err)
-	}
+	userjwt, err := GenerateJWTForUser(&user.User)
```

This err is not handled?

---

Done. Added handling code.